addons/mrp_subonctracting_landed_costs/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Landed Costs With Subcontracting order',
+    'version': '1.0',
+    'summary': 'Advanced views to manage landed cost for subcontracting orders',
+    'description': """
+This module allows add some search and details in order to manage landed cost for
```

Comment:
```suggestion
This module allows to add some searches and details in order to manage landed cost for
```
I think :thinking:

Reply:
I would write : 'This module allows users to more easily identify subcontracting orders when applying landed costs, by also displaying the associated picking reference in the search view.'